jstz-dev/jstz
# Jstzd: add rollup build script and spawn rollup in jstzd

# Context

[spawn rollup in jstzd](https://linear.app/tezos/issue/JSTZ-164/spawn-octezrollupnode-in-jstzd)

NOTE:
The jstz kernel rollup seems to not process the block in the CI (i suppose some issue with `nix.flake`)
the test works fine with the default nix settings `nix flake check`.
I also replaced the jstz kernel with the toy rollup kernel and that worked with the `nix.flake` 



# Description

There are two parts in this PR:
1. add a build script to build the rollup preimages / installer
a. Modified the MakeFile so that the kernel.wasm is copied to crates/jstzd/
b. Add build.rs that 1) stores the rollup kernel_installer.hex, preimages/ and parameters_ty.json to OUT_DIR. 2) generate a path getter file (jstz_rollup_path.rs) to access the path of 1
3. add rollup to the jstzd server


# Manually testing the PR
added rollup to the jstzd integration test.
```
make build
cargo test --package jstzd --test jstzd_test
```
```
nix flake check  
```
